src/test/java/com/fivetran/agent/mysql/output/OutputSpec.java

```diff
@@ -129,6 +153,14 @@ public void writeOut_alwaysWriteOnFinish() {
         } catch (Exception e) {
             throw new RuntimeException(e);
         }
-        assertFalse(mockOutputFile.isEmpty());
+        assertFalse(mockOutputFiles.isEmpty());
+    }
+
+    private String getDataFile() {
+        for (File f : mockOutputFiles.keySet()) {
+            if (f.getPath().contains("mysql_data"))
```

Comment:
It seems like we would also want to write tests affirming the content of mysql_state.json, right?